lib/base-compiler.js

```diff
@@ -316,15 +323,28 @@ export class BaseCompiler {
         const args = ['-d', outputFilename, '-l', ...objdumper.widthOptions];
         if (demangle) args.push('-C');
         if (intelAsm) args.push(...objdumper.intelAsmOptions);
-        const execOptions = {maxOutput: maxSize, customCwd: result.dirPath || path.dirname(outputFilename)};
 
-        const objResult = await this.exec(this.compiler.objdumper, args, execOptions);
-        if (objResult.code !== 0) {
-            result.asm = `<No output: objdump returned ${objResult.code}>`;
+        if (this.externalparser) {
+            result = await this.externalparser.objdumpAndParseAssembly(
+                result.dirPath,
+                args,
+                filters);
+            if (result.parsingTime !== undefined) {
+                result.objdumpTime = parseInt(result.execTime) - parseInt(result.parsingTime);
```

Comment:
Is there a reason why the `execTime` and `parsingTime` aren't already numbers?

Reply:
The idea was that it might be possible to include the unit of time, but it's never been anything other than milliseconds. Should probably convert everything to numbers.

Reply:
Got it. If we're in json we're already parsing, I guess if you later want to support it one could `result.execTime.value` and `result.execTime.units` if it's not type string.